tips/TIP-0048/openapi3-core.yaml

```
@@ -0,0 +1,3520 @@
+            Identifier of the output encoded in hex. An output is identified by
+            the concatenation of `transactionId+outputIndex` where `outputIndex` (u16) needs to be converted to little endian first.
+            Hex-encoded with 0x prefix.
+          example: "0xfa0de75d225cca2799395e5fc340702fc7eac821d2bdd79911126f131ae097a20100"
+      responses:
+        '200':
+          description: "Successful operation."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OutputMetadataResponse'
+              examples:
+                unspent:
+                  $ref: '#/components/examples/get-output-metadata-by-id-response-unspent-example'
+                spent:
+                  $ref: '#/components/examples/get-output-metadata-by-id-response-spent-example'
+        '400':
+          description: "Unsuccessful operation: indicates that the provided data is invalid."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BadRequestResponse'
+        '403':
+          description: "Unsuccessful operation: indicates that the endpoint is not available for public use."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ForbiddenResponse'
+        '404':
+          description: "Unsuccessful operation: indicates that the requested data was not found."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/NotFoundResponse'
+        '500':
+          description: "Unsuccessful operation: indicates that an unexpected, internal server error happened which prevented the node from fulfilling the request."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InternalErrorResponse'
+  '/api/core/v3/outputs/{outputId}/full':
+    get:
+      tags:
+        - UTXO
+      summary: Returns metadata about an output and the output structure by its identifier.
+      description: Returns the output structure and metadata by its identifier.
+      parameters:
+        - in: path
+          name: outputId
+          schema:
+            type: string
+          required: true
+          description: >-
+            Identifier of the output encoded in hex. An output is identified by
+            the concatenation of `transactionId+outputIndex` where `outputIndex` (u16) needs to be converted to little endian first.
+            Hex-encoded with 0x prefix.
+          example: "0xfa0de75d225cca2799395e5fc340702fc7eac821d2bdd79911126f131ae097a20100"
+      responses:
+        '200':
+          description: "Successful operation."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OutputWithMetadataResponse'
+              examples:
+                unspent:
```

The example is for a spent output
```suggestion
                spent:
```